modules/netns-isolation.nix

```diff
@@ -68,117 +75,88 @@ in {
         };
       });
     };
+
+    allowedUser = mkOption {
+      type = types.str;
+      description = ''
+        User that is allowed to execute commands in the service network namespaces.
+        The user's group is also authorized.
+      '';
+    };
+
+    netns = mkOption {
```

Comment:
Why is this needed?

Reply:
You mean the `netns` option?
See [netns: move webindex config for modules-only usage](https://github.com/fort-nix/nix-bitcoin/pull/226/commits/e1d60fdeea3fcc5e9bd5985fdd17db7d8912652f). It's used by `nix-bitcoin-webindex` to access the nginx netns address.

Reply:
Ah ok, so this exposes network namespace addresses to all modules? Very useful.